app/lib/shared/task_scheduler.dart

```diff
@@ -49,37 +44,16 @@ class ManualTriggerTaskSource extends TaskSource {
     });
   }
 
+  @override
+  Stream<Task> startStreaming() => _controller.stream;
```

Comment:
I believe that this is still insufficient for a certain kind of flooding attack, but overall it is a very low risk and low impact if it happens, keeping it simple is just fine.